src/scry/log.cr

```diff
@@ -1,13 +1,19 @@
 require "log"
 require "./client"
+require "logger"
+require "log/entry"
 
 module Scry
   module Log
     class_property logger : ::Log = ::Log.for("")
```

Comment:
I don't think this is needed anymore, I'd suggest renaming the current `Scry::Log` to `Scry::Logging` (or even simply put the `ClientLogger` in `Scry` directly).

Then we can have `Scry::Log` be the logger instance (from `::Log.for("scry")`)

Reply:
nice .